examples/complete/config/opsgenie-resources/api_integrations.yaml

```diff
@@ -0,0 +1,4 @@
+api_integrations:
```

Comment:
Let's not use these kinds of examples which we need to maintain as we upgrade our modules. They are similar enough to what we have elsewhere, but they detract from this module. This module is not about opsgenie, it's about yaml configurations. So let's just have some basic configurations that are not associated with other technologies.